dev/com.ibm.ws.app.manager.springboot/src/com/ibm/ws/app/manager/springboot/util/SpringBootThinUtil.java

```diff
@@ -478,6 +481,16 @@ public static StarterFilter getStarterFilter(Stream<String> entries) {
         return new StarterFilter(springBootStarter, starterArtifactIds);
     }
 
+    private static String extractVersion(String path) {
+        String version = "";
+        String regex = "(\\d+\\.\\d+\\.+\\d+)";
```

Comment:
Please ad a comment on what this regex does.

Reply:
done